ui/src/components/header/TranslationMenu.vue

```diff
@@ -18,36 +18,35 @@
 <template>
   <a-dropdown>
     <span class="action ant-dropdown-link translation-menu">
-      <font-awesome-icon :icon="['fas', 'language']" size="lg" />
+      <TranslationOutlined />
     </span>
-    <a-menu
-      slot="overlay"
-      :selectedKeys="[language]"
-      @click="onClick">
-      <a-menu-item key="en" value="enUS">English</a-menu-item>
-      <a-menu-item key="hi" value="hi">हिन्दी</a-menu-item>
-      <a-menu-item key="ja_JP" value="jpJP">日本語</a-menu-item>
-      <a-menu-item key="ko_KR" value="koKR">한국어</a-menu-item>
-      <a-menu-item key="zh_CN" value="zhCN">简体中文</a-menu-item>
-      <a-menu-item key="ar" value="arEG">Arabic</a-menu-item>
-      <a-menu-item key="ca" value="caES">Catalan</a-menu-item>
-      <a-menu-item key="de_DE" value="deDE">Deutsch</a-menu-item>
-      <a-menu-item key="es" value="esES">Español</a-menu-item>
-      <a-menu-item key="fr_FR" value="frFR">Français</a-menu-item>
-      <a-menu-item key="it_IT" value="itIT">Italiano</a-menu-item>
-      <a-menu-item key="hu" value="huHU">Magyar</a-menu-item>
-      <a-menu-item key="nl_NL" value="nlNL">Nederlands</a-menu-item>
-      <a-menu-item key="nb_NO" value="nbNO">Norsk</a-menu-item>
-      <a-menu-item key="pl" value="plPL">Polish</a-menu-item>
-      <a-menu-item key="pt_BR" value="ptBR">Português brasileiro</a-menu-item>
-      <a-menu-item key="ru_RU" value="ruRU">Русский</a-menu-item>
-      <a-menu-item key="el_GR" value="elGR">Ελληνικά</a-menu-item>
```

Comment:
Any particular reason this translation was removed?

Reply:
added